H/HiGHS/highs_common.jl

```diff
@@ -18,7 +18,8 @@ sources = [
 platforms = supported_platforms()
 
 function build_script(; shared_libs::String)
-    return "BUILD_SHARED=$(shared_libs)\n" * raw"""
+    build_static = shared_libs == "OFF" ? "ON" : "OFF"
+    return "BUILD_SHARED=$(shared_libs)\nBUILD_STATIC=$(build_static)\n" * raw"""
 cd $WORKSPACE/srcdir/HiGHS
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/HiGHS

# Remove system CMake to use the jll version
apk del cmake
```

I forgot that we also have to remove the system CMake to get it to use the JLL version :man_facepalming:.